app/views/partials/steps.html

```diff
@@ -2,10 +2,11 @@
 <ul class="nav nav-pills" id="steps-nav">
   <li role="presentation" ng-repeat="step in steps"
     ng-class="{active: step.id == currentStep.id, disabled: !step.isCurrent && !step.isPassed}">
-    <a ng-href="{{ step.isPassed || step.isCurrent ? step.route : 'javascript:void(0)' }}">
+    <a href='#'
+       ng-click="goToStep(step)">
```

Comment:
It is fairly common to see breaks like this in framework code (Angular, React) for readability, line length, etc.